ci/build-unix.yml

```
@@ -67,7 +67,7 @@ steps:
     condition: and(succeeded(),
                    eq(${{parameters.is_release}}, 'true'),
                    eq(variables['Build.SourceBranchName'], 'main'),
-                   eq('${{parameters.name}}', 'linux'))
+                   or(eq('${{parameters.name}}', 'linux'), eq('${{parameters.name}}', 'linux-scala-2.13')))
```

I think you can use [the `in` function](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml#run-for-continuous-integration-ci-builds-if-succeeding) here.

```suggestion
                   in('${{parameters.name}}', 'linux', 'linux-scala-2.13'))
```